ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+ToolbarDelegate.swift

```diff
@@ -260,41 +260,43 @@ extension BrowserViewController: TopToolbarDelegate {
       return handleIPFSSchemeURL(url)
     }
 
-    if let url = URL(string: text), url.scheme == "brave" {
+    if let url = URL(string: text), url.scheme == "brave" || url.scheme == "chrome" {
```

Comment:
Eh maybe we can move this to a helper method like IsBraveScheme or something similar

Reply:
I'll be deleting this line. I didn't realize it snuck through. It was added for testing only.